### PR TITLE
Thermodynamics

### PR DESCRIPTION
**Pull request by Julia-Pl1**
Please review my code in file 'funkcja1.cpp' as well.

### CODE REVIEW

File: funkcja_termodynamika_1/funkcja1.cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#include <tgmath.h>
+#include "constants.h"
+
+int main ()
+{
+  std::cout << "The first law of thermodynamics is also known as The Law of Conservation of Energy" << std::endl;
+  double q;
+  std::cout << "Type how much heat was supplied to the system:" << std::endl;
+  std::cin >> q;
+  std::cout << "Type how much work was done on the layout:" << std::endl;
```
Comment (miklimek) [inline funkcja_termodynamika_1/funkcja1.cpp:13]: The word 'layout' sounds a bit weird in the context. Why not just write 'work done by system' or 'work done on/by gas'?

**Review by miklimek — APPROVED** [review]
Aside from the wording I think it's all right as a first function

File: funkcja_termodynamika_1/funkcja1.cpp (hunk 1 above)
Comment (PaulinaGacek) [reply funkcja_termodynamika_1/funkcja1.cpp:13]: I would opt for "work done by/on the system" as it is the most commonly used phrase in my opinion

File: funkcja_termodynamika_1/funkcja1.cpp [hunk 2]
```diff
@@ -13,20 +13,15 @@ void TheFirstLaw()
   std::cout << "Type how much work was done on the layout:" << std::endl;
   double w;
   std::cin >> w;
```
Comment (PaulinaGacek) [inline funkcja_termodynamika_1/funkcja1.cpp:15]: Didn't you forget about cin>>q ? Because I cannot find it in your code.

Comment (miklimek) [reply funkcja_termodynamika_1/funkcja1.cpp:15]: It's in line 12

**Comment by Julia-Pl1** [comment]
I will change the word 'layout' as it was indicated. Thank you

**Pull request merged**